component/wallet-cli/cmd/oidc4vp.go

```diff
@@ -76,6 +90,17 @@ func createFlags(cmd *cobra.Command) {
 	cmd.Flags().String("oidc-client-id", "", "oidc client id. example: test-org")                                                              //nolint
 	cmd.Flags().String("oidc-client-secret", "", "oidc client secret. example: test-org-secret")                                               //nolint
 	cmd.Flags().Bool("skip-schema-validation", false, "skip schema validation for while creating vp")                                          //nolint
+	cmd.Flags().Bool("oidc4-vp-should-request-credentials", true, "indicates if oidc4vp flow should request new credentials")
+
+	cmd.Flags().BoolVar(&flags.InsecureTls, "insecure-tls", false, "disable certificate validation for http client")
+
+	cmd.Flags().StringVar(&flags.WalletUserId, "wallet-user-id", "", "existing wallet user id")
+	cmd.Flags().StringVar(&flags.WalletPassPhrase, "wallet-passphrase", "", "existing wallet pass phrase")
+	cmd.Flags().StringVar(&flags.StorageProvider, "storage-provider", "", "storage provider. supported: mem,mysql,mongodb")
```

Comment:
For a CLI, it would be nice to _also_ have a persistent database option that doesn't a running service.
@DRK3 @skynet2 @fqutishat

Reply:
done @troyronda